docs/docs/getting-started-devs.md

```diff
@@ -72,12 +113,20 @@ If you aren't using a big RDF library like Jena or RDF4J, the simplest way to ge
 
 ```xml
```

Comment:
Tabs please! :) You can also add sbt.

If you are so inclined, you can even generate these via an mkdocs macro defined in main.py. Up to you, I'm fine with either.